drf_roles/mixins.py

```diff
@@ -19,7 +18,14 @@ class RoleError(Exception):
 class RoleViewSetMixin(object):
     """A ViewSet mixin that parameterizes DRF methods over roles"""
     _viewset_method_registry = set(getattr(settings, "VIEWSET_METHOD_REGISTRY", DEFAULT_REGISTRY))
-    _role_groups = set(getattr(settings, "ROLE_GROUPS", DEFAULT_GROUPS))
+
+    def __init__(self, **_kwargs):
```

Comment:
Why `**_kwargs`?

Reply:
It is the [`initkwargs`](https://github.com/encode/django-rest-framework/blob/master/rest_framework/viewsets.py#L96) param passed when constructing the `ViewSet`.  If this is not included then the process crashes with:

```
  File "./lib/python3.6/site-packages/rest_framework/viewsets.py", line 96, in view
    self = cls(**initkwargs)
TypeError: __init__() got an unexpected keyword argument 'suffix'
```